config/packages/easy_admin.yaml

```
@@ -0,0 +1,62 @@
+easy_admin:
+  site_name: 'NNIZER'
+  user:
+    name_property_path: 'name'
+    display_avatar: true
+
+  list:
+    title: 'List of %%entity_label%%'
+    actions:
+      - { name: edit, icon: pencil, label: Edit }
+      - { name: delete, icon: exclamation, label: Delete }
+      - { name: search, label: Search }
+      - { name: new, icon: plus, label: Add new  }
+
+  entities:
+    User:
+      class: App\Entity\User
+      list:
+        fields:
+          - 'id'
+          - 'email'
+          - 'roles'
+          - 'name'
+          - 'createdAt'
+      form:
+        fields:
+          - 'email'
+          - 'name'
```

```suggestion
          - 'name'
          - { property: 'roles', type: 'choice', type_options: { multiple: true, choices: { 'ROLE_USER': 'ROLE_USER', 'ROLE_ADMIN': 'ROLE_ADMIN' } } }
```
Add this line to show list of Roles to choose